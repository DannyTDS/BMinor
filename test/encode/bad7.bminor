"Hello World!\0x"