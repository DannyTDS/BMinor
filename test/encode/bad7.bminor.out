Error: failed to read \0xHH hex value.
Cannot decode string "Hello World!\0x".
